### ISSUE TITLE
Abholen und Abladen erzeugt neue Ziele

### ISSUE THREAD

**Issue by fliiix69** [Stephan-S/FS22_AutoDrive#198]
Guten Abend,
uns ist leider schon öfter aufgefallen, dass der Modus Abholen und Abladen bei uns auf dem Dedicated Server zu Problemen führt.
Ordnerstruktur ist angeschaltet in AD. Wählen wir nun unsere Ziele aus und starten AD, so springt das oberste Ziel "Abladen" um.
Dadurch entsteht ein neues Ziel, z.B. "Molkerei" im Standard Ordner und AD fährt diesen an. Der Punkt ist dann vor meinem eigentlich selbst gesetzten Ziel "Molkerei" einfach ein paar Punkte davor/danach. Lösche ich das neu angelegte Ziel dann wieder und wähle wieder den Modus Abholen und Abladen (Beispiel: Abholen -> Kuhstall ... Abliefern -> Schokofabrik) so wird wieder ein neuer Zielpunkt erstellt und der Punkt "Abholen" springt auf diesen um und fährt dort hin..

Ist dies ein Bug/Feature, oder sind wir einfach nur etwas unfähig?

**Comment by SaschaLie** [comment]
Hast Du evtl. die kleinen Symbole Abladen/Abholen grün? Dann sucht sich AD das Ziel mit den Besten Preisen aus und erstellt neue Punkte. Ist ein Feature. Klick die Symbole an, damit sie nicht mehr grün sind

**Comment by fliiix69** [comment]
> Hast Du evtl. die kleinen Symbole Abladen/Abholen grün? Dann sucht sich AD das Ziel mit den Besten Preisen aus und erstellt neue Punkte. Ist ein Feature. Klick die Symbole an, damit sie nicht mehr grün sind

Vielen Dank für dein Typ, ich habe meine Leute mal darauf sensibilisiert. Wir werden das mal im Auge behalten und berichten =)

**Comment by Modyne** [comment]
Hallo! Habe das gleiche Problem. Spiele im Singleplayer, Problem ist reproduzierbar:
Hab mir in Haute Beyleron Wege zur Standart-BGA eingefahren, genau zu den Abkipp-Punkten für Gülle und Silage. Stelle AD auf Abholen->Gülle am Kuhstall + Abkippen->BGA-Gülleplatz. Solange die Symbole weiß sind fährt er da hin. Sobald man auf Grün umstellt kreiert es in der BGA (und am Kuhstall, wenn das auch auf Grün umgestellt wird) einen neuen Punkt. Aber den NICHT am Platz wo man abkippen kann bzw wo er Gülle nachtanken kann. Der neue Punkt liegt zwar innerhalb des BGA-Geländes (bzw Kuhstall-Geländes) und auf meinem eingefahrenen Weg, aber nicht am Abkipp/Auffüll-Punkt.

Laut Symbolbeschreibung (weiß-grün) haben die meines Erachtens nichts mit "besten Preisen" zu tun sondern sollen doch lediglich justieren ob AD automatisch be-/entladen dürfen oder der Trekker dort waren soll. (bitte gern korrigieren, wenn ich hier falsch liege).

Hoffe, dies ist ein Bug, der bald behoben wird...

**Comment by haamic** [comment]
@Modyne 
Und warum stellst du auf grün? bzw. was stellst du auf grün? grün gibts doch gar nicht mehr. welche alte version nutzt du denn?
Meinst du die Logos vor Start und Ziel? die sollten immer weiss sein, wenn du nichts teuer verkaufen, oder günstig einkaufen willst. 
anderenfalls draufklicken (sind dann hellblau nicht mehr grün) und AD sucht sich das entsprechende Ziel und erstellt ggf. einen Wegpunkt, wenn keiner vorhanden ist.

Vielleicht sind deine Zielpunkte zu weit weg und AD meint es gäbe noch keine, erzeugt deshalb nachträglich noch welche. Ich weiss es nicht. Screenshot davon wäre eine feine Sache und würde den Entwicklern helfen, das evtl. korrigieren zu können.

**Comment by Modyne** [comment]
Sorry, blau meine ich. Ja, die Logos vor Start und Ziel. Version ist die 2.0.0.3
Vielleicht ist hier der englische Tooltipp für die Funktion nicht aussagekräftig genug. Ich hab´s so verstanden, dass durch das umschalten eine automatische Be- und Entladung passiert. bei deaktivierter (=weißem Icon) Funktion der Fahrer wartet. Hab´s aber bisher nie getestet.

Hier im Screenshot sieht man den manuell erstellten Entladepunkt, rechts weiter hinten auf der Wegstrecke den automatisch vom AD erstellten punkt.

![fsScreen_2022_01_17_18_02_32](https://user-images.githubusercontent.com/97887354/149812605-cd401dd6-d156-480d-a3ac-2ba37e0b55d6.png)

**Comment by fliiix69** [comment]
> Sorry, blau meine ich. Ja, die Logos vor Start und Ziel. Version ist die 2.0.0.3 Vielleicht ist hier der englische Tooltipp für die Funktion nicht aussagekräftig genug. Ich hab´s so verstanden, dass durch das umschalten eine automatische Be- und Entladung passiert. bei deaktivierter (=weißem Icon) Funktion der Fahrer wartet. Hab´s aber bisher nie getestet.
> 
> Hier im Screenshot sieht man den manuell erstellten Entladepunkt, rechts weiter hinten auf der Wegstrecke den automatisch vom AD erstellten punkt.
> 
> ![fsScreen_2022_01_17_18_02_32](https://user-images.githubusercontent.com/97887354/149812605-cd401dd6-d156-480d-a3ac-2ba37e0b55d6.png)

Drück mal vorne auf das Symbol mit dem gekippten Hänger im AD Menü, sodass dieses weiß ist anstatt blau.
Wie im zweiten Post erklärt sucht er sich dann irgendwelche Ziele zum Abladen. Auch bei mir hat er wie oben beschrieben immer wieder neue Ziele erstellt. Seit ich nun darauf geachtet habe, dass die Symbole immer weiß sind und nicht grün/blau je nach Version werden keine neuen Ziele erstellt und er fährt wie er soll.

**Comment by Modyne** [comment]
Ja, hab ich bereits gemacht. Das war sicherlich mein Fehler und mein AD tut jetzt so wie´s soll.

Aber die Funktion an sich ist ja hier scheinbar buggy, AD sollte bei aktivierter Funktion sicherlich automatisch keine neuen Punkte erstellen, an denen das Produkt nicht entladen werden kann.
